man/man1/fzf.1

```diff
@@ -1278,6 +1278,15 @@ e.g.
      \fBfzf --bind space:jump,jump:accept,jump-cancel:abort\fR
 .RE
 
+\fIheader-click\fR
```

Comment:
I would name it `click-header` because we may add more `click-xxx` events in the future and it would be nice for them to share the same `click-` prefix.